primitives/Cargo.toml

```diff
@@ -12,6 +13,7 @@ sp-runtime = { branch = "rococo-v1", default-features = false, git = "https://gi
 [features]
 default = ["std"]
 std = [
+    "arbitrary",
```

Comment:
`arbitrary` is a development dependency that will be shipped on production builds. So, it is necessary to remove this line, add `#[cfg(feature = "arbitrary")]` to `primitives/src/types.rs` and then also add `zeitgeist-primitives = { features = ["arbitrary"], path = "../../../primitives" }` to the fuzz project.

It is also possible to manually construct `MultiHash` inside the fuzz project.